### DISCUSSION TITLE
Custom 500 page

### DISCUSSION THREAD

**Discussion by tommhuth** [vercel/next.js#19000] [Help]
Following Nextjs own documentation I've added a `pages/_error.js` and when I trigger an exception on purpose in one of my pages for test sake locally, the error page renders as expected, however it does so without any styles except styles specific for the error page (meaning my footer and header for instance are completely styleless, using SCSS modules). Am I missing something here? Is there any limitations to `_error.js`? Am I right in expecting it to function like a React error boundary?

Also, when I publish to Netlify, the error does not render and I instead get an ugly text output like:

```
{"errorType":"Runtime.UnhandledPromiseRejection","errorMessage":"TypeError: _src_blocks_video_list_VideoList__WEBPACK_IMPORTED_MODULE_2__.a.s is not a function" }
```

**Comment by lfades** [comment]
There shouldn't be any limitations with `pages/_error.js`, however it's not expected to work like an error boundary. If your page fails to render or throws then the error page will be used instead. I am unsure about how Netlify handles this but you should try it out in your machine using `next build && next start` and the result there is the one Next.js handles (in development we use a custom error page to display errors so test it on production mode with a build)

Provide more details and maybe even a reproduction that we can take a look at 🙏 . Right now this is not a bug.
